packages/components/src/components/BackButton/BackButton.stories.mdx

```diff
@@ -0,0 +1,44 @@
+import {
+  Canvas,
+  Story,
+  Meta,
+  ArgsTable,
+  PRIMARY_STORY,
+} from '@storybook/addon-docs';
+import { BackButton } from '.';
+import {
+  Source,
+  ComponentLinkRow,
+  SB_DESIGNSYSTEM_URL,
+  LinkToInteractiveStory,
+} from '../../storybook';
+
+<Meta title="Design system/BackButton/API" component={BackButton} />
+
+# BackButton
+
+<ComponentLinkRow
+  docsHref=""
```

Comment:
Her er docslenke du kan bruke 😄 

```suggestion
  docsHref="https://design.domstol.no/987b33f71/p/72c4bc-backlink"
```

Reply:
Ohh my god, jeg satt jo å tenkte at jeg måtte huske å kalle komponenten for `BackLink` 😂 Fiksa nå!